<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SideBar</class>
 <widget class="QWidget" name="SideBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QWidget" name="list_extend" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #282828;
border-radius: 10px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>1</number>
          </property>
          <item alignment="Qt::AlignLeft">
           <widget class="QPushButton" name="list_icon_extend">
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/resources/menu.png</normaloff>:/resources/menu.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #bb86fc;</string>
            </property>
            <property name="text">
             <string> Quiz App</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QPushButton" name="home_extend_button">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  color: #bb86fc;
}

QPushButton:hover {
	background: #DCDCDC;
 }

QPushButton:checked {
	background: #DCDCDC;
 }</string>
          </property>
          <property name="text">
           <string>  Home</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/resources/home.png</normaloff>:/resources/home.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QPushButton" name="quizes_extend_button">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  color: #bb86fc;
}

QPushButton:hover {
	background: #DCDCDC;
 }

QPushButton:checked {
	background: #DCDCDC;
 }
</string>
          </property>
          <property name="text">
           <string>  Quizes</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/resources/quiz.png</normaloff>:/resources/quiz.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QPushButton" name="results_extend_button">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  color: #bb86fc;
}

QPushButton:hover {
	background: #DCDCDC;
 }

QPushButton:checked {
	background: #DCDCDC;
 }</string>
          </property>
          <property name="text">
           <string>  Results</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/resources/results.png</normaloff>:/resources/results.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="quit_extend_button">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  color: #bb86fc;
}

QPushButton:hover {
	background: #DCDCDC;
 }</string>
          </property>
          <property name="text">
           <string>  Quit</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/resources/exit.png</normaloff>:/resources/exit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="list_combact" native="true">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #282828;
border-radius: 10px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="list_icon_compact">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #bb86fc;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/resources/menu.png</normaloff>:/resources/menu.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="home_compact_button">
          <property name="styleSheet">
           <string notr="true">QPushButton {
  color: #bb86fc;
}

QPushButton:hover {
	background: #DCDCDC;
 }

QPushButton:checked {
	background: #DCDCDC;
 }</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/resources/home.png</normaloff>:/resources/home.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="quizes_compact_button">
          <property name="styleSheet">
           <string notr="true">QPushButton {
  color: #bb86fc;
}

QPushButton:hover {
	background: #DCDCDC;
 }

QPushButton:checked {
	background: #DCDCDC;
 }</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/resources/quiz.png</normaloff>:/resources/quiz.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="results_compact_button">
          <property name="styleSheet">
           <string notr="true">QPushButton {
  color: #bb86fc;
}

QPushButton:hover {
	background: #DCDCDC;
 }

QPushButton:checked {
	background: #DCDCDC;
 }</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/resources/results.png</normaloff>:/resources/results.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="quit_compact_button">
          <property name="styleSheet">
           <string>QPushButton {
  color: #bb86fc;
}

QPushButton:hover {
	background: #DCDCDC;
 }</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/resources/exit.png</normaloff>:/resources/exit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>list_icon_compact</sender>
   <signal>pressed()</signal>
   <receiver>list_extend</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_icon_compact</sender>
   <signal>pressed()</signal>
   <receiver>list_combact</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_icon_extend</sender>
   <signal>pressed()</signal>
   <receiver>list_combact</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_icon_extend</sender>
   <signal>pressed()</signal>
   <receiver>list_extend</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
